### DISCUSSION TITLE
What is the reason that you store literature notes notes in Obsidian and do you actually link them within the same vault as your permanent notes?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#27] [Q&A] [answered]
> Hi Bryan, I am trying to implement Obsidian and the Zettelkasten system in my daily workflow. As a PhD candidate I have to read quite a lot, but I often find this effort quite unfruitful because I will forget everything I read after a while. Your video was the first to actually present a complete overview of a workflow, that was super helpful! I have one question though. In his book, Ahrens writes that you should store your literature notes in a different folder altogether (like in a reference system), so they won't really impact your slip-box directly. What is the reason that you store these notes in Obsidian and do you actually link them within the same vault as your permanent notes?
-- Andries Peeters

**Comment by tallguyjenks** [comment, marked as answer]
my taxonomy system for notes does separate them in my vault so in that respect they follow that suggestion. 

I keep them in the vault so I can easily reference where potential idea notes originated from or were inspired by. 

how it looks from a birds-eye view might be: 

1. I journal in my daily note where I also mention `[[that paper I read today]]` 
2. `[[that paper I read today]]` contains literature notes for that paper from my research paper workflow
3. those literature notes inspire ideas which I link from the paper and use my tagging system see my [Tag Taxonomy](https://publish.obsidian.md/bryan-jenks/Tag+Taxonomy) 

so the linking might looks like this (real example from my vault):

- `[[2020-09-09]]`
    + ![image](https://user-images.githubusercontent.com/29872822/105401661-4778da00-5bdb-11eb-88d3-f6c951360e7c.png)
- `[[{ 2020-09-09 How To Take Smart Notes]]`
    + ![image](https://user-images.githubusercontent.com/29872822/105401908-90c92980-5bdb-11eb-93c8-67fcaab3d560.png)
- `[[Well defined tasks negate the need for the use of the limited supply of will power]]`
    + ![image](https://user-images.githubusercontent.com/29872822/105401847-7a22d280-5bdb-11eb-93c0-f744db16ea44.png)

**Reply by Aipeeter** [reply]
Thank you very much for the quick and very thorough answer!

So in the end your literature notes will only link to the ideas they sparked (and your journal of course) and only these ideas will be linked to other ideas to create the meaningful links that are necessary for the system to work?

**Reply by tallguyjenks** [reply]
@Aipeeter bingo! :)

**Reply by bsoist** [reply]
@tallguyjenks this is something I'm still a little fuzzy about - and the whole "inbox" idea.

Would you mind sharing what tags these three example notes have once this process is completed for these three notes?

**Reply by tallguyjenks** [reply]
@bsoist my daily notes dont get a tag, the literature note for the book once done is 📥/📚/🟩 and the note after that is 📝/🌲
